lib/kuality-kfs/data_objects/purap/001_item_line_object.rb

```diff
@@ -108,6 +102,28 @@ def edit_extended_attributes(attribute_group=nil)
     # proper symbols.
   end
 
+
+  def determine_commodity_code_of_type(commodity_type='non-sensitive')
+    # Cannot randomly select out of all the options available on the sensitive data dropdown
+    # because not all options have data associated with them. AFT will fail when no data provided.
+    # Dropdown is zero based. Positions zero, one, and four have data values associated to them.
+    # Position zero returns any non-sensitive commodity code represented by this method's return
+    # value of an empty string.
+    # As of March 2016, positions one and four return commodity codes defined as sensitive.
+    # In order for AFTs to be able to randomly use either value, an odd-even randomization
+    # was implemented with DEFAULTS_FOR_ITEMS odd_commodity and even_commodity used respectively.
+    set_dropdown_to = ''
+    if not(commodity_type.eql?('non-sensitive'))
```

Comment:
[Use `!` instead of `not`](http://stackoverflow.com/questions/27429711/can-someone-explain-not-vs-in-ruby)